src/utils/frame.js

```diff
@@ -95,6 +99,11 @@ export function getLibraryFromUrl(frame: Frame) {
     return "jQuery";
   }
 
+  // Needs to remain before "react", otherwise "react" can also match "preact"
```

Comment:
We could make react more specific by maybe changing the react regex to `/^react/i`

Reply:
If I'm not mistaken, we check the entire url, which means `react` can be anywhere in the url and not just in the beginning. We can try to get the filename from the url and then it would be easier.

Reply:
aaaahhhh ... my bad ! ur right. what of this `/(react)/i` instead

Reply:
It would create a capture group, but still match `react` as part of `preact`.

Reply:
The closest thing I found is reverse look-ahead (use look-ahead after reversing the string) but I think it's too complex.

Reply:
https://regexr.com/3gt7c